src/web/mjs/engine/Chapter.mjs

```diff
@@ -89,6 +89,7 @@ export default class Chapter extends EventTarget {
      * and a reference to the page list (undefined on error).
      */
     getPages( callback ) {
+        document.dispatchEvent(new CustomEvent(EventListener.onChapterGetPages, { detail: this }));
```

Comment:
This will fire every time chapter starts downloading.
May use the UI preview button click event instead to fire the event.

Reply:
This is intended, since I personally never read a manga in HakuNeko I would like to state my latest 'activity'.